### PR TITLE
Updated firebase analytics to a newer version

### PR DESCRIPTION
**Pull request by ruialmeida51**
Updated the firebase analytics and the way we track a page hit.

### CODE REVIEW

**Review by ritatferreira — APPROVED** [review]

File: analytics-firebase/src/main/java/com/mindera/skeletoid/analytics/appenders/FbAppender.kt [hunk 1]
```diff
@@ -53,6 +58,20 @@ class FbAppender : IAnalyticsAppender {
         firebaseAnalytics?.setCurrentScreen(activity, screenName, screenClassOverride)
     }
 
+    override fun trackPageHit(screenClass: String, screenName: String) {
+        if (firebaseAnalytics == null) {
+            LOG.e(LOG_TAG, "trackPageHit failed: firebaseAnalytics is null")
+            return
+        }
+
+        val parameters = mapOf(
```
Comment (bruno-garces) [inline analytics-firebase/src/main/java/com/mindera/skeletoid/analytics/appenders/FbAppender.kt:67]: Can't we use `bundleOf` ?

File: analytics-firebase/src/test/java/com/mindera/skeletoid/analytics/appenders/FbAppenderUnitTest.kt [hunk 2]
```diff
@@ -141,31 +141,53 @@ class FbAppenderUnitTest {
     fun testTrackPageHit() {
         val activity = Activity()
         val screenName = "screenName"
+        val screenClass = "ScreenActivity"
 
         appender.trackPageHit(activity, screenName, null)
 
         verify(firebaseAnalytics).setCurrentScreen(activity, screenName, null)
+
+        val screenTrackPayload = Bundle().apply {
```
Comment (bruno-garces) [inline analytics-firebase/src/test/java/com/mindera/skeletoid/analytics/appenders/FbAppenderUnitTest.kt:150]: `bundleOf` ?

File: base/src/main/java/com/mindera/skeletoid/analytics/appenders/interfaces/IAnalyticsAppender.kt [hunk 3]
```diff
@@ -52,6 +52,17 @@ interface IAnalyticsAppender {
...
+    /**
+     * Track app page hit manually
+     *
+     * @param screenClass         Screen class name
+     * @param screenName          Screen name
+     */
+    fun trackPageHit(
+        screenClass: String,
+        screenName: String
+    )
```
Comment (bruno-garces) [inline base/src/main/java/com/mindera/skeletoid/analytics/appenders/interfaces/IAnalyticsAppender.kt]: Single line?

**Review by bruno-garces — CHANGES_REQUESTED** [review]

File: analytics-firebase/src/main/java/com/mindera/skeletoid/analytics/appenders/FbAppender.kt (hunk 1 above)
Comment (ruialmeida51) [reply analytics-firebase/src/main/java/com/mindera/skeletoid/analytics/appenders/FbAppender.kt:67]: we do not have the required imports for that on this module

File: base/src/main/java/com/mindera/skeletoid/analytics/appenders/interfaces/IAnalyticsAppender.kt (hunk 3 above)
Comment (ruialmeida51) [reply base/src/main/java/com/mindera/skeletoid/analytics/appenders/interfaces/IAnalyticsAppender.kt]: sure 👍

**Review by bruno-garces — APPROVED** [review]

**Pull request merged**